src/components/styles/Book.css

```diff
@@ -1,54 +1,58 @@
-/* Book container */
+@import '~typeface-playfair-display';
+
 .book {
   display: flex;
   flex-direction: column;
-  background-color: #f9f9f9;
-  border: 1px solid #ddd;
-  padding: 1rem;
-  margin-bottom: 1rem;
-  border-radius: 4px;
+  padding: 10px;
+  flex-wrap: wrap;
+  margin-right: 20px;
 }
 
-/* Category */
 .category {
-  font-size: 0.8rem;
-  text-transform: uppercase;
-  margin-bottom: 0.5rem;
+  color: rgb(142, 140, 140);
+  padding-left: 35px;
+  margin-bottom: 5px;
 }
 
-/* Book details */
 .book-details {
+  font-size: 1.5rem;
+  font-weight: 800;
   display: flex;
   flex-direction: column;
-  flex: 1;
 }
 
-/* Book name */
 .book-name {
-  font-size: 1.1rem;
-  font-weight: bold;
-  margin-bottom: 0.2rem;
+  font-family: 'Playfair Display', serif;
+  font-size: 2.5rem;
+  font-weight: 900;
+  color: black;
+  margin-bottom: 5px;
+  padding-left: 35px;
 }
```

Comment:
- [ ] If you see the requirement then `your final design should match Zeplin's design (this includes all elements + the same styling)`. So please stick to the design as much as possible like color, font-size, etc. You can compare your design with original deisgn.
- [ ] - [ ] Note: You can use . [Login to Zeplin](https://app.zeplin.io/login). providing the password and email that microverse provided. From there you can use the same color, font, font family and so on. 
### Your design
![his](https://github.com/Xghayor/Book-store/assets/81727988/23a9f3ac-2a78-4462-b3a3-87bacbe76ab7)

### Original design
![original](https://github.com/Xghayor/Book-store/assets/81727988/c23a5747-40f0-4ee6-b55f-fa67d1dcd59c)